### PR TITLE
Refactor/styles2

### PR DESCRIPTION
**Pull request by Karlfunhouse**
Dialed in the styling for the second game box on the right side for the Names & Guesses.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,37 +21,49 @@ <h4 class="range-input">Set Range</h4>
...
+                <label class="guess" for="guess2">Guess</label>
+                <input class="game-input guess" id="guess2" type="number" name="player">
+              </section>
+            </section>
             <section class="button-box">
-              <label for="submit-button" class="hide">SUBMIT GUESS</label>
+              <label for="submit-button" class="hide button-text">SUBMIT GUESS</label>
               <input type="button" id="submit-button" disabled class="play-input game-button" name="" value="SUBMIT GUESS">
-              <label for="reset-button" class="hide">RESET GAME</label>
+              <label for="reset-button" class="hide button-text">RESET GAME</label>
```
Comment (KmBarnett) [inline index.html]: Why did we add the class button-text to labels that are hidden with the hide class they aren't being shown on the page

Comment (KmBarnett) [reply index.html]: As well as I don't see button-text in the css?

**Review by KmBarnett — CHANGES_REQUESTED** [review]
I just don't see the need for the button-text i think we should take it out unless you used it in another file?

File: index.html (hunk 1 above)
Comment (Karlfunhouse) [reply index.html]: Yeah, I was trying to style the button text but didn't get it to work quite yet.  I'll just cut that out for now.

**Pull request merged**